detectron2/data/detection_utils.py

```diff
@@ -24,6 +24,17 @@
 from . import transforms as T
 from .catalog import MetadataCatalog
 
+convert_image_with_exif = {
+    1: lambda img: img,
+    2: lambda img: img.transpose(Image.FLIP_LEFT_RIGHT),                              # horizontal flip
+    3: lambda img: img.transpose(Image.ROTATE_180),                                   # 180 degree rotate
+    4: lambda img: img.transpose(Image.FLIP_TOP_BOTTOM),                              # vertical flip
+    5: lambda img: img.transpose(Image.FLIP_LEFT_RIGHT).transpose(Image.ROTATE_90),  # horizontal flip & counter clock 90 degree rotate
+    6: lambda img: img.transpose(Image.ROTATE_270),                                   # counter clock wise 270 degree rotate
+    7: lambda img: img.transpose(Image.FLIP_LEFT_RIGHT).transpose(Image.ROTATE_270), # horizontal flip & counter clock wise 270 degree rotate
+    8: lambda img: img.transpose(Image.ROTATE_90),                                    # counter clock wise 90 degree rotate
+}
+
```

Comment:
according to https://stackoverflow.com/questions/13872331/rotating-an-image-with-orientation-specified-in-exif-using-python-without-pil-in we can require PIL >=6.0 and use
```
from PIL import ImageOps

image = ImageOps.exif_transpose(image)
```

Reply:
Thanks! I will refactor near future.